app/controllers/api/v1/workflows_controller.rb

```diff
@@ -72,7 +72,9 @@ def extract_strings(tasks)
 
   def add_relation(resource, relation, value)
     if relation == :retired_subjects && value.is_a?(Array)
+      resource.save!
```

Comment:
the `add_relation` method here should only be called via the `update_links` so the only state change here is retiring the subject. Pretty sure this would be ok to reload as we've got no other attributes to change here.